### ISSUE TITLE
Fix ARIA errors

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5509]
Fix errors involving ARIA attributes.

- **Error/Serious:** [ARIA hidden element must not be focusable or contain focusable elements](https://dequeuniversity.com/rules/axe/4.8/aria-hidden-focus?application=axeAPI)
  Ensures aria-hidden elements are not focusable nor contain focusable elements

- **Error/Critical:** [Elements must only use supported ARIA attributes](https://dequeuniversity.com/rules/axe/4.8/aria-allowed-attr?application=axeAPI)
  Ensures an element's role supports its ARIA attributes
